### PR TITLE
Add support for TeamCity

### PR DESCRIPTION
**Pull request by rbuussyghin**
This pull request adds basic support for TeamCity CI

### CODE REVIEW

File: lib/danger/ci_source/teamcity.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      end
+
+      def supported_request_sources
+        @supported_request_sources ||= [Danger::RequestSources::GitHub]
+      end
+
+      def initialize(env)
+        self.repo_slug       = env['GITHUB_REPO_SLUG']
+        self.pull_request_id = env['GITHUB_PULL_REQUEST_ID'].to_i
+        self.repo_url        = env['GITHUB_REPO_URL']
```
Comment (rbuussyghin) [inline lib/danger/ci_source/teamcity.rb]: Unfortunately, TeamCity doesn't export GitHub specific info via environment variables. So you have to add these environment variables manually to your project or build configuration.

**Comment by rbuussyghin** [comment]
> @rbuussyghin is not a contributor yet, would you like to join the Danger org?
@DangerCI no

**Comment by orta** [comment]
Looks good, could you add TeamCity to the README, and add a note in the source code about having to do it manually, I think we'll need to add this in the docs once the site is ready 👍

**Comment by rbuussyghin** [comment]
@orta done. Any ETA on next release?

**Comment by orta** [comment]
There wasn't too much going on, so it feels reasonable to ship this as 0.8.2

**Comment by rbuussyghin** [comment]
Oh, that's great. I'm in the middle of setting up Danger for my project and I can't wait for the next release with TeamCity support!

**Pull request merged**